src/program/snabb_lwaftr/tests/data/binding.table

```diff
@@ -1,7 +1,13 @@
 {
- {'127:2:3:4:5:6:7:128', '178.79.150.233', 1, 100, '8:9:a:b:c:d:e:f'},
- {'127:11:12:13:14:15:16:128', '178.79.150.233', 101, 64000},
- {'127:22:33:44:55:66:77:128', '178.79.150.15', 5, 7000},
- {'127:24:35:46:57:68:79:128', '178.79.150.2', 7800, 7900, '1E:1:1:1:1:1:1:af'},
- {'127:14:25:36:47:58:69:128', '178.79.150.3', 4000, 5050, '1E:2:2:2:2:2:2:af'}
+ {'127:2:3:4:5:6:7:128', '178.79.150.233', {80, 0, 16}, '8:9:a:b:c:d:e:f'},
+ {'127:11:12:13:14:15:16:128', '178.79.150.233', {2300, 0, 16}},
```

Comment:
here can we specify these as psid=X, shift=S, length=X, with shift defaulting to 16-length ?

The positional names are too hard to read, and I think naming them A and K is not so great either.  I argue that "shift" and "length" are better names -- to get the PSID from a port, you shift right by SHIFT, and then take the lowest LENGTH bits.  With LENGTH taking its default value, we get contiguous port ranges.

Probably internally we should work on SHIFT and LENGTH instead of A and K.  The change is irritating, and I know this must be irritating to you.  I apologize.  What do you think about the suggestion though?  We could of course transform SHIFT/LENGTH into A/K, but to me there's no value in that.  I always forget which one is which.  That could just be me though.

If you agree, the format should be then e.g.

```
{'127:11:12:13:14:15:16:128', '178.79.150.233', {psid=2700, shift=0}},
```

or whatever.